travis-ci/cleanup-integration-tests.sh

```diff
@@ -4,31 +4,37 @@ set -e
 
 . ./travis-ci/set-env-vars.sh
 
-cd example || exit 1
-
-# Delete the stack if it's a nightly build
-if [ "$DEPLOYMENT" = "cumulus-nightly" ]; then
-  yarn
-  echo Delete app deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder app \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete iam deployment
-
-  ./node_modules/.bin/kes cf delete \
-    --kes-folder iam \
-    --region us-east-1 \
-    --deployment "$DEPLOYMENT" \
-    --yes
-
-  echo Delete app deployment
-else
-  yarn add @cumulus/common
-fi
-
-echo Unlocking stack
-node ./scripts/lock-stack.js false $DEPLOYMENT
\ No newline at end of file
+(
+  set -e
+
+  cd example
+
+  rm -rf node_modules
```

Comment:
Why are we doing this? Because `npm ci` is going to do it?

Reply:
This goes back to the strange behavior that I was seeing yesterday.  If you run `lerna bootstrap`, then run `npm install` in the `example` directory, you get all sorts of errors about missing packages.  I was never able to figure out _why_ that was happening, but did figure out that deleting `example/node_modules` before running `npm install` in the `example` directory prevented those errors.

Reply:
Does this happen locally or only in Travis?

Reply:
I was able to reproduce the broken behavior in a docker container running locally, I don't know if I tested it directly on my laptop.

Reply:
I just tested it locally and it didn't seem to happen

Reply:
I'm running a test without this now.

Reply:
But it's not going to get run until CMR comes back: https://travis-ci.org/nasa/cumulus/builds/508020484

Reply:
Looks like it has the same problem as the others, in that there's no simple-git to require in the setting of the env vars.  so something will have to be done here.  Here's the result. https://travis-ci.org/nasa/cumulus/jobs/508020491